compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Check.hs

```diff
@@ -880,6 +880,14 @@ checkIface m iface = do
     forM_ (intChoices iface) (checkTemplateChoice tcon)
     checkExpr (intPrecondition iface) TBool
 
+  -- check view method exists
+  case NM.lookup (MethodName "_view") (intMethods iface) of
```

Comment:
I thought I had left a comment here but it looks like it disappeared somewhere: Do you really want to special case the `_view` method? I used that in the PoC as a hack until we have proper interface views but my expectation was that in the final version we include this as a separate definition in `DefInterface` and don’t hijack a magic method name.

Reply:
Our thinking was that we get the checks working first, then add syntax to remove the special-casing it in a subsequent PR, to keep PRs manageable

Reply:
makes sense, just wanted to be clear on the expected state at the end.